conda.recipe/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: pystog
+  version: 1.0.0
```

Comment:
PyStoG is on version 0.2.6 (see [PyPi release history](https://pypi.org/project/pystog/#history) for PyStoG)

Shouldn't have to hard-code the version, we should pick it up from tags in the repository

Reply:
The chunk of code found [here](https://github.com/neutrons/mantid_total_scattering/blob/master/conda.recipe/mantid-total-scattering-python-wrapper/meta.yaml#L1-L5) accomplishes that.

Reply:
Done